src/ts/relayer.ts

```diff
@@ -55,24 +56,25 @@ export class RelayerAPI {
     const callData = this.getForwardCallData(relayer, params);
 
     return relayer.sendTransaction({
-      to: params.hub, // Potentially a ProxyAccount
+      to: params.to, // Potentially a ProxyAccount
       data: callData,
     });
   }
 
   /**
-   * If it is a ProxyHub, we will fetch the signer's contract account address.
+   * If it is a ProxyAccountFactory, we will fetch the signer's contract account address.
```

Comment:
Elsewhere you call this ProxyAccountDeployer -choose one and replace all others

Reply:
replaced